src/network/networkd-network-gperf.gperf

```diff
@@ -195,6 +195,7 @@ IPv6Prefix.PreferredLifetimeSec,        config_parse_prefix_lifetime,
 CAN.BitRate,                            config_parse_si_size,                           0,                             offsetof(Network, can_bitrate)
 CAN.SamplePoint,                        config_parse_permille,                          0,                             offsetof(Network, can_sample_point)
 CAN.RestartSec,                         config_parse_sec,                               0,                             offsetof(Network, can_restart_us)
+CAN.TripleSampling,                     config_parse_bool,                              0,                             offsetof(Network, can_triple_sampling)
```

Comment:
Please try to use `config_parse_tristate()` if it is applicable, and initialize `can_triple_sampling` with `-1`.